web/web.go

```diff
@@ -51,23 +50,23 @@ import (
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/common/route"
 	"github.com/prometheus/common/server"
-	"github.com/prometheus/prometheus/tsdb"
-	"github.com/prometheus/prometheus/tsdb/index"
-	"github.com/soheilhy/cmux"
-	"golang.org/x/net/netutil"
-	"google.golang.org/grpc"
-
 	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/notifier"
 	"github.com/prometheus/prometheus/promql"
 	"github.com/prometheus/prometheus/rules"
 	"github.com/prometheus/prometheus/scrape"
 	"github.com/prometheus/prometheus/storage"
 	"github.com/prometheus/prometheus/template"
+	"github.com/prometheus/prometheus/tsdb"
+	"github.com/prometheus/prometheus/tsdb/index"
 	"github.com/prometheus/prometheus/util/httputil"
 	api_v1 "github.com/prometheus/prometheus/web/api/v1"
 	api_v2 "github.com/prometheus/prometheus/web/api/v2"
 	"github.com/prometheus/prometheus/web/ui"
+	"github.com/soheilhy/cmux"
+	"go.uber.org/atomic"
+	"golang.org/x/net/netutil"
+	"google.golang.org/grpc"
```

Comment:
Not `prometheus/prometheus` :) so let's keep it above and separated for consistency

Reply:
Hopefully finally addressed in 568bd51 🙏